Sources/Hummingbird/Server/RequestID.swift

```diff
@@ -15,7 +15,7 @@
 import Atomics
 
 /// Generate Unique ID for each request
-public struct RequestID: CustomStringConvertible {
+public struct RequestID: CustomStringConvertible, Sendable {
```

Comment:
Don't know why but the compiler only just started complaining about this now, but it is a valid complaint